### PR TITLE
add options

### PR DESCRIPTION
**Pull request by krsntn**
allow to specify commit's date start & commits on an initiallized repo

### CODE REVIEW

**Comment by Shpota** [comment]
Hi @krsntn !

Thank you for your contribution. And I'm sorry for the late response (at the moment, I am busy with my other project - [goxygen](https://github.com/Shpota/goxygen)).

I'll review the changes soon.

File: contribute.py [hunk 1]
```diff
@@ -85,6 +95,12 @@ def arguments():
                         to the repository. The link is accepted in SSH or HTTPS
                         format. For example: git@github.com:user/repo.git or
                         https://github.com/user/repo.git""")
+    parser.add_argument('-sd', '--start_date', type=int, default=366,
+                        required=False, help="""Defines the start date of
+                        commit""")
+    parser.add_argument('-u', '--update', action='store_true', default=False,
+                        required=False, help="""Wheather is to update existing
```
Comment (eduncan911) [inline contribute.py:102]: """Rather to update existing

**Comment by eduncan911** [comment]
When I run this version of the script, I get an error when passing in `--update` or `-u`:

```
$ python2 ./contribute.py --max_commits=20 --frequency=68 --start_date=30 -u
Traceback (most recent call last):
  File "./contribute.py", line 108, in <module>
    main()
  File "./contribute.py", line 24, in main
    run(['git', 'clone', repository])
  File "./contribute.py", line 59, in run
    Popen(commands).wait()
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
TypeError: execv() arg 2 must contain only strings
```

**Review by Monishka11 — COMMENTED** [review]
Nice work

**Comment by Shpota** [comment]
This functionality is implemented in the latest version.

**Pull request closed without merging**